src/threatmodels/threatmodel.js

```diff
@@ -166,13 +166,14 @@ function threatModel($scope, $location, $routeParams, dialogs, common, dataconte
 
     function duplicateDiagram(index) {
         var duplicatedDiagram = angular.copy(vm.threatModel.detail.diagrams[index]);
-        vm.newDiagram.title = "Copy of " + duplicatedDiagram.title;
-        vm.newDiagram.id = newID();
-        vm.newDiagram.diagramJson = duplicatedDiagram.diagramJson;
-        vm.newDiagram.size = duplicatedDiagram.size;
-        vm.threatModel.detail.diagrams.push(vm.newDiagram);
-        vm.newDiagram = emptyDiagram();
```

Comment:
This would have caused duplicated LINDDUN and CIA diagrams to always be of type STRIPE....